src/ray/object_manager/pull_manager.cc

```diff
@@ -131,7 +134,7 @@ void PullManager::Tick() {
     const auto time = get_time_();
     if (time >= request.next_pull_time) {
       TryPull(object_id);
-      request.next_pull_time = time + pull_timeout_ms_ / 1000;
+      request.num_retries++;
```

Comment:
How about keeping the logic here instead of in TryPull?

Reply:
I think @stephanie-wang prefers to update the retry time after every pull, not just when a retry is fired.

Reply:
Yeah I thought it would be better to do it here because sometimes we call `TryPull` from other places.

Reply:
Sounds good.